Source/WebCore/platform/graphics/cocoa/SourceBufferParserWebM.cpp

```diff
@@ -1250,10 +1253,11 @@ webm::Status WebMParser::AudioTrackData::consumeFrameData(webm::Reader& reader,
             return Skip(&reader, bytesRemaining);
         }
 
-        m_packetDuration = MediaTime(formatDescription->framesPerPacket, formatDescription->rate);
-
+        m_packetDurationParser = makeUnique<PacketDurationParser>(*formatDescription);
         setFormatDescription(formatDescription.releaseNonNull());
-    } else if (codec() == CodecType::Opus) {
+    }
+#if !HAVE(AUDIOFORMATPROPERTY_VARIABLEPACKET_SUPPORTED)
+    else if (codec() == CodecType::Opus) {
         // Opus technically allows the frame duration and frames-per-packet values to change from packet to packet.
         // CoreAudio doesn't support ASBD values like these to change on a per-packet basis, so throw an error when
```

Comment:
Comment out of date. And as it turns out, it was never correct in the first place. It wasn't anything general about CoreAudio that imposed the constraint but the codec instead.